src/operator/nn/cudnn/cudnn_pooling-inl.h

```diff
@@ -73,15 +73,17 @@ class CuDNNPoolingOp {
     CUDNN_CALL(cudnnDestroyPoolingDescriptor(pooling_desc_));
   }
 
-  void Forward(const OpContext &ctx, const TBlob &in_data,
+  // Return boolean saying whether pooling configuration is supported.
+  bool Forward(const OpContext &ctx, const TBlob &in_data,
```

Comment:
Is this boolean return necessary? I don't find it in other operators. What happens if it returns false?